views/docs/partials/sections/post-root.pug

```diff
@@ -0,0 +1,23 @@
+section#post-root
+  h2 Route: / 
+  p Example URL - <a href='http://localhost:1990/'>http://localhost:1990/</a>
```

Comment:
This claims `/` as the route but it is really `/id` which is what the side nav says. Watch your consistency.